client/src/components/ui/Accordion.js

```diff
@@ -0,0 +1,7 @@
+import { Component } from 'react';
+
+class Accordion extends Component {
```

Comment:
this looks like a very important component

Reply:
lol

Reply:
maybe i'll extract it later

Reply:
what its not even used